File: StoryFlow_Shared/Models/GetWordLessonDto.cs
namespace StoryFlow_Shared.Models
{
    public class GetWordLessonDto
    {
        public int Id { get; set; }
        public int? OrderInSeason { get; set; }
        public int? StorySeasonId { get; set; }
        public string? PolishTitlte { get; set; }
        public string? Image { get; set; }
        public List<GetWordDto> Words { get; set; } = new();
        public GetWordPointDto? WordPoint { get; set; }
        public List<GetUserWordLessonDto> UserWordLessons { get; set; } = new();
    }
}

File: StoryFlow_Client/src/app/models/get-word-lesson-dto.ts
import { GetUserWordLessonDto } from "./get-user-word-lesson-dto";
import { GetWordDto } from "./get-word-dto";
import { GetWordPointDto } from "./get-word-point-dto";

export interface GetWordLessonDto {
    id: number;
    orderInSeason: number | null;
    storySeasonId: number | null;
    polishTitlte: string | null;
    image: string | null;
    words: GetWordDto[];
    userWordLessons: GetUserWordLessonDto[];
    wordPoint: GetWordPointDto | null;
}